docs/versioned_docs/version-2.11.0/introduction/features.mdx

```diff
@@ -0,0 +1,79 @@
+---
+id: features
+title: OpenEBS Features
+---
+
+import { TwoColumn } from "../../../src/components/TwoColumn";
+
+- [Containerized storage for containers](#containerized-storage-for-containers)
+- [Synchronous replication](#synchronous-replication)
+- [Snapshots and clones](#snapshots-and-clones)
+- [Backup and restore](#backup-and-restore)
+- [Prometheus metrics and Grafana dashboard](#prometheus-metrics-for-workload-tuning)
+
+:::tip
+For information on how OpenEBS is used in production, visit the [use cases](/docs/introduction/usecases) section or see examples of users and their stories on the OpenEBS Adopters section [here](https://github.com/openebs/openebs/blob/master/ADOPTERS.md) where you can also share your experience.
+:::
+
+### Containerized Storage for Containers
+
+<TwoColumn>
+  <p>
+    OpenEBS2 is an example of Container Attached Storage or CAS. Volumes
+    provisioned through OpenEBS are always containerized. Each volume has a
+    dedicated storage controller that increases the agility and granularity of
+    persistent storage operations of the stateful applications. Benefits and
+    more details on CAS architecture are found <a href="/docs/next/cas.html">here</a>.
+  </p>
+
+![Containerized Storage Icon](../assets/f-cas.svg)
```

Comment:
is it possible to have image names? 
``` 
![Containerized Storage Icon](../assets/f-cas.svg)
```
something else or the same as the alt text ??

Reply:
Also, check for other places if images are been updated.

Reply:
@AVRahul  -- this is all coming from old website of docs.

Reply:
Oh ok, I was a bit confused by the image name. 
Anyway, it's ok as it is coming from old docs.